src/ucp/rndv/rndv_ppln.c

```diff
@@ -245,8 +255,16 @@ static ucs_status_t ucp_proto_rndv_ppln_progress(uct_pending_req_t *uct_req)
             return UCS_OK;
         }
 
+        /* Avoid sending message less than `frag_proto_min_length` */
```

Comment:
why is it needed?

Reply:
See explanation here: https://github.com/openucx/ucx/pull/9728#discussion_r1522764256

We can discuss it in more details if needed.